rotkehlchen/data/importer.py

```diff
@@ -297,7 +298,54 @@ def _consume_cryptocom_entry(self, csv_row: Dict[str, Any]) -> None:
                 link='',
             )
             self.db.add_asset_movements([asset_movement])
-
+        elif row_type in ('airdrop_to_exchange_transfer', 'mco_stake_reward'):
+            asset = symbol_to_asset_or_token(csv_row['Currency'])
+            amount = deserialize_asset_amount(csv_row['Amount'])
+            action = LedgerAction(
+                identifier=0,  # whatever is not used at insertion
+                timestamp=timestamp,
+                action_type=LedgerActionType.INCOME,
+                location=Location.CRYPTOCOM,
+                amount=amount,
+                asset=asset,
+                rate=None,
+                rate_asset=None,
+                link=None,
+                notes='',
```

Comment:
notes is also optional.
```suggestion
                notes=None,
```